manifests/l/Lazarus/Lazarus/3.8/Lazarus.Lazarus.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.4.3 $debug=QUSU.CRLF.7-4-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Lazarus.Lazarus
+PackageVersion: "3.8"
+InstallerLocale: en-US
+InstallerType: inno
+AppsAndFeaturesEntries:
+- Publisher: Lazarus Team
+  ProductCode: '{code:GetAppId}_is1'
```

```suggestion
  ProductCode: lazarus_is1
```

@Trenly - Or shall I change it to match the ARP Table from `SandboxTest.ps1` inspection? :pray: